app/src/androidTest/java/app/cybrid/demoapp/ui/kyc/IdentityVerificationActivityTest.kt

```diff
@@ -39,13 +36,14 @@ class IdentityVerificationActivityTest {
 
     private lateinit var resources: Resources
 
-    @Before
+    //@Before
```

Comment:
?

Reply:
Do you remember that I gonna turn off the e2e testing for Indentity Verification Component!